tests/from_meta.rs

```diff
@@ -64,3 +64,55 @@ fn nested_meta_lit_bool_errors() {
         Error::unsupported_format("literal").to_string()
     );
 }
+
+/// Tests behavior of FromMeta implementation for enums (including behavior of defaults).
+mod enum_impl {
+    use darling::{Error, FromMeta};
+    use syn::parse_quote;
+
+    /// A playback volume.
+    /// Deriving `Default` will cause the "default" variant to be set when the meta-item is empty.
+    #[derive(Debug, Clone, Copy, PartialEq, Eq, Default, FromMeta)]
+    #[darling(default)]
+    enum Volume {
+        #[default]
```

Comment:
I think the question of how to handle just the word form is discussed in #63; given that this isn't truly a default, I'd prefer to use the `#[darling(word)]` syntax rather than a bare `default` here.